### ISSUE TITLE
kserve-container torchserve env setting problem

### ISSUE THREAD

**Issue by juhyeon-cha** [kserve/kserve#2571]
/kind bug

**What steps did you take and what happened:**
kserve-container for pytorch does not have TS_SERVICE_ENVELOPE env value

**What did you expect to happen:**
TS_SERVICE_ENVELOPE env must have "kserve" value

**What's the InferenceService yaml:**
inferenceService yaml is very simple and not special

**Anything else you would like to add:**
The problem occurred when I changed the code in handler.py
I'm not sure if this is the cause of the problem.

#### Error
```python
import torch

from ts.torch_handler.base_handler import BaseHandler

class Handler(BaseHandler):
    def preprocess(self, data):
        return torch.FloatTensor(data[0])

    def postprocess(self, data):
        return [torch.argmax(data, dim=0).tolist()]//EOF

```

#### Normal
```python
import torch

from ts.torch_handler.base_handler import BaseHandler

class Handler(BaseHandler):
    def preprocess(self, data):
        return torch.FloatTensor(data[0])

    def postprocess(self, data):
        return [torch.argmax(data, dim=0).tolist()]
//EOF
```

#### kserve controller manager log
```json
{
  "level": "info",
  "ts": 1669678291.3029177,
  "logger": "PredictorReconciler",
  "msg": "Resolved container",
  "container": "&Container{Name:kserve-container,Image:kserve/torchserve-kfs:0.5.3,Command:[],Args:[torchserve --start --model-store=/mnt/models/model-store --ts-config=/mnt/models/config/config.properties],WorkingDir:,Ports:[]ContainerPort{},Env:[]EnvVar{EnvVar{Name:TS_SERVICE_ENVELOPE,Value:,ValueFrom:nil,},},Resources:ResourceRequirements{Limits:ResourceList{cpu: {{1 0} {<nil>} 1 DecimalSI},memory: {{2147483648 0} {<nil>} 2Gi BinarySI},},Requests:ResourceList{cpu: {{1 0} {<nil>} 1 DecimalSI},memory: {{1073741824 0} {<nil>} 1Gi BinarySI},},},VolumeMounts:[]VolumeMount{},LivenessProbe:nil,ReadinessProbe:nil,Lifecycle:nil,TerminationMessagePath:,ImagePullPolicy:,SecurityContext:nil,Stdin:false,StdinOnce:false,TTY:false,EnvFrom:[]EnvFromSource{},TerminationMessagePolicy:,VolumeDevices:[]VolumeDevice{},StartupProbe:nil,}",
  "podSpec": {
    "containers": [
      {
        "name": "kserve-container",
        "image": "kserve/torchserve-kfs:0.5.3",
        "args": [
          "torchserve",
          "--start",
          "--model-store=/mnt/models/model-store",
          "--ts-config=/mnt/models/config/config.properties"
        ],
        "env": [
          {
            "name": "TS_SERVICE_ENVELOPE"
          }
        ],
        "resources": {
          "limits": {
            "cpu": "1",
            "memory": "2Gi"
          },
          "requests": {
            "cpu": "1",
            "memory": "1Gi"
          }
        }
      }
    ]
  }
}
```

**Environment:**

- Istio Version: 1.14.1
- Knative Version: 1.2.5
- KServe Version: 0.8.0
- Kubeflow version: 1.6.1
- Kubernetes version: 1.21.6

**Comment by yuzisun** [comment]
@juhyeon-cha Did you follow the tutorial [here](https://kserve.github.io/website/0.8/modelserving/v1beta1/torchserve/#creating-model-storage-with-model-archive-and-config-file) ?

**Comment by yuzisun** [comment]
@juhyeon-cha the environment variable should be injected by default, seems like for some reason it is missing the value. 

https://github.com/kserve/kserve/blob/48146646d76a133da09bf65f1c84f69df9e2a0e6/pkg/apis/serving/v1beta1/inference_service_defaults.go#L257

https://github.com/kserve/kserve/blob/7baebda3ea4b2d856550e77eb0ee6e36da5d6fc5/install/v0.8.0/kserve-runtimes.yaml#L180

**Comment by juhyeon-cha** [comment]
@yuzisun I don't know the details of why the problem is happening

I solved the problem by putting the values ​​directly into the labels when serving the pytorch framework
sorry for not providing more details
